src/test/Calculator.test.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import Calculator from '../components/Calculator';
+
+it('renders correctly', () => {
+  const tree = render(<Calculator />);
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] Merely importing functions (like `render`) from the React Testing Library (`@testing-library/react`) is not enough. As requested previously, there should be tests that simulate user actions like events. You are yet to add these. Kindly add them.

  You can refer to the article I shared earlier for hints.

Reply:
Noted

Reply:
I have read the article and I have added a simulation for the user interaction in the calculate.js file. Can you kindly help look at it before I submit it for final review? this is our last chance. Thanks

Reply:
- Hey @vickymarz, I believe one of your tests is failing (see image).

- Also, you added the simulation test in the `calculate.test.js` file, whereas I highlighted the **`Calculator.test.js`** file.

  ![vik-3](https://user-images.githubusercontent.com/36542986/161297595-240a3fc4-f601-4e5c-89f3-444e4e6c6c40.jpg)

Reply:
wow. Do you mean I should add this exact line of code in Calculator.test.js?

Reply:
THank you for your response. it means a lot

Reply:
@vickymarz Yeah. It would be more organised that way. In the `calculate.test.js` file you are testing math logic.

In the `Calculator.test.js` file, you are testing the component itself, which has the buttons and events and so on. Hence it would make sense to add them there.

Reply:
We have added them there. Can you help us checkout or probably assist us please 🙏

Reply:
@samiullahbahadur I gave the initial requirements, which you are expected to follow.

These subsequent suggestions are merely MY opinion for better code organisation. You can simply write the React Testing Library tests (however you wish) as I initially requested.

Reply:
@TedLivist what means this import { render } from '@testing-library/react';
it's not React Testing Library?

Reply:
Hey @samiullahbahadur merely **RENDERING** a component with RTL is not the same as **SIMULATING USER ACTIONS** on the component with RTL. After rendering you still need to simulate actions as stated in the requirements.